spec/spec_helper.rb

```diff
@@ -1,87 +1,12 @@
 require 'coveralls'
+require 'omniauth'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.